px-tooltip.html

```diff
@@ -852,13 +922,21 @@
     _getClass: function() {
       var res = 'tooltip-container ';
 
-      if(!this._isShowing) {
+      if(!this.visible) {
         res += ' hidden ';
       }
 
       //style the arrow
       res += this._tooltipPos;
       return res;
+    },
+
+    _ignoreTargetEventsChanged: function() {
```

Comment:
To get this Polymer 2 ready (since its new code), maybe take `ignoreTargetEvents` as an argument from the observer and check if it is defined then use it in the function?

Reply:
it has a default value so will always be defined, so I think it's fine. We should probably check for undefined only when we know something might not be defined